qiskit_ibm_runtime/api/session.py

```diff
@@ -51,20 +54,33 @@
 
 def _get_client_header() -> str:
     """Return the client version."""
-    try:
-        client_header = "qiskit/" + pkg_resources.get_distribution("qiskit").version
-        return client_header
-    except Exception:  # pylint: disable=broad-except
-        pass
-
-    qiskit_pkgs = ["qiskit-terra", "qiskit-aer", "qiskit-ignis", "qiskit-aqua"]
-    pkg_versions = {"qiskit-ibm-runtime": ibm_runtime_version}
+
+    qiskit_pkgs = [
+        "qiskit-terra",
+        "qiskit-aer",
+        "qiskit-experiments",
+        "qiskit-nature",
+        "qiskit-machine-learning",
+        "qiskit-optimization",
+        "qiskit-finance",
+    ]
+
+    pkg_versions = {
+        "qiskit-ibm-runtime": "{}-{}".format("qiskit-ibm-runtime", ibm_runtime_version)
+    }
     for pkg_name in qiskit_pkgs:
         try:
-            pkg_versions[pkg_name] = pkg_resources.get_distribution(pkg_name).version
+            version_info = "{}-{}".format(
```

Comment:
you could also use an f-string here. I don't know which way is more pythonic, though.